general-concepts/licenses/text.xml

```diff
@@ -82,11 +82,38 @@ versions are specified elsewhere.
 </p>
 
 <p>
-Please watch for license conflicts. If the license indicated
-by the package is incompatible with the licenses used by its sources
-(e.g. BSD/MIT package including GPL sources), please contact
-the licenses team for guidance. Do not add packages that seem
-to include license term violations.
+If an installed file is built from sources using different licenses, all those
+licenses must be compatible. The effective license for the file will be the most
+restrictive of input licenses. For example:
+</p>
+
+<ol>
+	<li>
+		If a program includes both GPL- and BSD-licensed code, the license
+		for the resulting program will be respective GPL license since it is more
+		restrictive than BSD.
+	</li>
+	<li>
+		If a program includes 'GPL-2 or newer' and 'GPL-3 [or newer]' code,
+		the latter will be effective as it is the subset of both terms.
+	</li>
+	<li>
+		If a program includes 'GPL-2 or newer' and 'GPL-2' code, 'GPL-2' will
+		be the effective license.
+	</li>
+</ol>
+
+<p>
+Please watch out for license conflicts. If the package combines incompatible
+licenses (e.g. GPL-2 and GPL-3 code without 'or newer' clauses), please contact
+the licenses team for guidance. Do not add packages that seem to include
+license term violations.
+</p>
+
+<p>
+For more information, please see
+<uri link="https://www.gnu.org/licenses/license-compatibility.en.html">
+License Compatibility and Relicensing</uri> article.
```

Comment:
Maybe add that this only describes compatibility for the GPL family?

Reply:
I disagree. The first few paragraph cover different lax licenses as well.